src/content/en/tools/chrome-devtools/webauthn/index.md

```diff
@@ -0,0 +1,92 @@
+project_path: /web/tools/chrome-devtools/_project.yaml
+book_path: /web/tools/chrome-devtools/_book.yaml
+description: Emulate Authenticators and Debug WebAuthn in Chrome DevTools.
+
+{# wf_updated_on: 2020-09-27 #}
+{# wf_published_on: 2020-09-28 #}
+{# wf_blink_components: Security, Platform>DevTools #}
```

Comment:
Are these the same components as displayed in monorail? If so, instead of Security we should have Blink>WebAuthentication.